.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+This issue template is meant mainly for bug reports and feature requests.
+
+You can still use an issue to ask for help. In this case
+you should remove the template contents or use it as a guideline
+for providing information for debugging.
+-->
+
+### Kind of issue
+
+> Uncomment only one of these:
+> 
+> Bug
+> Feature request
+
+### Observed behavior
+
+
+### Expected/desired behavior
+
+
+### Details on how to reproduce (minimal and precise)
+
+
+### Information about the environment:
+
+- Glusterd2 version used (e.g. v4.1.0 or master): 
+- Operating system used: 
+- Glusterd2 compiled from sources, as a package (rpm/deb), or container: 
+- Using External ETCD: (yes/no, if yes ETCD version):
+- If container, which container image: 
+- Using kubernetes, openshift, or direct install: 
+- If kubernetes/openshift, is gluster running inside kubernetes/openshift or outside: 
+
+
+### Other useful information
+
+* glusterd2 config file
+* glusterd2 log file
+* ETCD configuration
```

Comment:
If it's an etcd issue, we'll likely need etcd logs from all nodes too

Reply:
they might not be knowing the whether its ETCD issue or not, we can get it later if we think its an ETCD issue